index.css

```diff
@@ -614,3 +614,173 @@ hr {
     border-radius: 2px;
   }
 }
+
+/* Modal style */
+
+.modal-card {
+  display: flex;
+  flex-direction: column;
+  padding: 4% 3%;
+  border-radius: 20px;
+  margin: 9% 6%;
+  gap: 1.2rem;
+  height: 100%;
+  border: 1px solid #dfe1e6;
+  background-color: #fff;
+}
```

Comment:
**[OPTIONAL]**

- [ ] Hey, each modal for mobile and desktop has a container and the container has a `background: #C1C7D0;`. as this wasn't pointed out by the previous reviewer, it's marked optional. You can implement it to improve your design. check the Figma design or check the images the previous reviewer added, you should see the background.